_docs/geyser/anticheat-compatibility.md

```diff
@@ -10,29 +10,30 @@ title: Anticheat Compatibility
 
 Full Compatibility (Checks Bedrock Players Accurately)
 
-- [AntiAura](https://www.spigotmc.org/resources/1368/){:target="_blank"} (Paid) - Last Updated on 19th August 2021
-- [Themis](https://www.spigotmc.org/resources/90766/){:target="_blank"} - Last Updated on 15th Jun 2022
-- [Negativity V2](https://www.spigotmc.org/resources/86874/){:target="_blank"} (Paid) - Last Updated on 21st March 2022
-- [SoaromaSAC](https://www.spigotmc.org/resources/87702/){:target="_blank"} - Last Updated on 27th March 2022
-- [Spartan: Bedrock Edition](https://builtbybit.com/resources/12832/){:target="_blank"} (Paid) - Last Updated on 22nd May 2023
+- [AntiAura](https://www.spigotmc.org/resources/1368/){:target="_blank"} (Paid) - Last Updated on 24th January 2023
+- [FoxAddition](https://www.spigotmc.org/resources/111260/){:target="_blank"} - Last Updated on 19th November 2023
+- [LiteAntiCheat](https://www.spigotmc.org/resources/112053/){:target="_blank"} - Last Updated on 26th August 2023
+- [Negativity V1](https://www.spigotmc.org/resources/48399/){:target="_blank"} - Last Updated on 31st August 2023
+- [Negativity V2](https://www.spigotmc.org/resources/86874/){:target="_blank"} (Paid) - Last Updated on 1st November 2023
+- [SoaromaSAC](https://www.spigotmc.org/resources/87702/){:target="_blank"} - Last Updated on 17th April 2023
+- [Spartan: Bedrock Edition](https://builtbybit.com/resources/12832/){:target="_blank"} (Paid) - Last Updated on 17th November 2023
+- [Themis](https://www.spigotmc.org/resources/90766/){:target="_blank"} - Last Updated on 27th October 2023
```

Comment:
I think the intention here was to show when the wiki page was updated, not when the plugin was